src/components/sdk/DeviceSelection/SpeakerSelection/index.tsx

```diff
@@ -26,7 +26,7 @@ export const SpeakerSelection: React.FC<Props> = ({
   const { devices, selectedDevice } = useAudioOutputs();
   const selectAudioOutput = useSelectAudioOutputDevice();
 
-  async function selectDevice(deviceId: string) {
+  async function selectDevice(deviceId: string): Promise<void> {
     selectAudioOutput(deviceId);
```

Comment:
Not  related to your change, but since `selectAudioOutput` also returns a Promise should we `await it?

Reply:
I have fixed this in my latest commit, so it will come with next PR. Would you mind not fixing it in this PR?

Reply:
Np at All!

Reply:
Thanks, but it's a small fix, so I updated it : )